hooks/pre/35-change-evr-extension-ownership.rb

```diff
@@ -0,0 +1,17 @@
+if local_postgresql?
+  response, _status = execute_command("sudo systemctl is-active postgresql", false, true)
+
+  unless response.equal? "active"
+    start_services(['postgresql'])
+  end
+
+  logger.notice("Updating ownership of the evr extension if it is enabled")
+  result = execute!("runuser -l postgres -c \"psql -d foreman -c \\\"SELECT 1 from pg_extension WHERE extname='evr';\\\"\"", false, true)
```

Comment:
Is this needed? I'd expect that the `UPDATE` line is simply a noop if it's not installed.

Reply:
Removed the checks..The query is a no-op if evr extension isn't found..